app/pages/search/SearchPage.js

```diff
@@ -96,6 +106,15 @@ class UnconnectedSearchPage extends Component {
             showMap={showMap}
           />
         )}
+
+        <div className="container">
```

Comment:
You don't put container in random place like this, either you make the whole component as a big container, or not using it at all

Reply:
Removed the div with the container and wrapped Sort and SearchResults with Page Wrapper in the recent commit.